pom.xml

```diff
@@ -205,8 +205,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>1.6</source>
-                    <target>1.6</target>
+                    <source>1.8</source>
+                    <target>1.8</target>
```

Comment:
I think if you plan to implement this functionality with Java 8 features, it's better to submit this pull request to the master branch (for the version 4.0.0 which supports Java 8). For the 3.2 branch we still need to maintain Java 6.